models/stg_klaviyo__campaign.sql

```diff
@@ -31,14 +31,15 @@ final as (
         from_email,
         from_name,
         cast(id as {{ dbt.type_string() }} ) as campaign_id,
-        is_segmented,
         name as campaign_name,
         send_time as scheduled_to_send_at,
         sent_at,
         coalesce(status, lower(status_label)) as status,
         status_id,
         subject,
         updated as updated_at,
+        archived,
```

Comment:
since this is a boolean do we wanna rename it to `is_archived`?

Reply:
Ah thanks Jamie-- just realized I had some changes that weren't pushed, this included